.github/workflows/test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
             fail-fast: false
             matrix:
                 laravel: [7.*, 8.*]
-                php: [7.2, 7.3, 7.4]
+                php: [7.2, 7.3, 7.4, 8.0]
                 service: ['mysql:8', 'mysql:5.7', mariadb]
 
                 include:
```

Comment:
Can we add an experimental tag to the include for PHP 8.0 and then instead of checking the specific PHP version for the composer dependency installs, check the experimental tag?

Reply:
I tried that route first... wasn't playing nicely with the other includes we have below, but I can try that again.  I agree it would certainly be cleaner.